charts/area-chart/step4.md

```diff
@@ -1,61 +1,34 @@
-Now we'll add multiple datasets to the area chart...
+Now you will add tooltips to the area chart.
 
 ## Task
 
-1) Make sure the App.js file is still open
+1) Make sure the App.js file is still open.

```

Comment:
Did we decide on making step titles bold or not? because we do this in other tutorials?